src/sentry/utils/pytest/sentry.py

```diff
@@ -265,6 +265,13 @@ def register_extensions():
     )
 
 
+def pytest_runtest_setup(item):
+    if not settings.MIGRATIONS_TEST_MIGRATE and any(
+        mark for mark in item.iter_markers(name="migrations")
+    ):
+        pytest.skip("migrations are not enabled, run with MIGRATIONS_TEST_MIGRATE=1 pytest ...")
```

Comment:
That's pretty nice, previously it'd fail cryptically if you ran a migration specific test